client-instance.go

```diff
@@ -10,28 +10,50 @@ import (
 	"github.com/SkycoinProject/dmsg"
 )
 
-var (
-	singleton *dmsg.Client
-	once      sync.Once
+// Defaults for dmsg configuration, such as discovery URL
+const (
+	DefaultDiscoveryURL = "http://dmsg.discovery.skywire.cc"
```

Comment:
I would not make the test deployment the default. For now it makes more sense, but as soon as Murphy updates prod, we should use prod.

Reply:
whats the prod url?

Reply:
"http://dmsg.discovery.skywire.skycoin.com"